### DISCUSSION TITLE
[Feedback]  Difficulty (phone number) joining the optimism Discord

### DISCUSSION THREAD

**Discussion by megtog** [ethereum-optimism/developers#298] [Feedback]
### Feedback

Hello I was trying to join the [the optimism Discord](https://discord.com/channels/667044843901681675/1080862392281481246)

I can not use my phone because my country calling code is not supported by discord.

![image](https://github.com/ethereum-optimism/developers/assets/106442058/d5a58ac0-ec8f-492a-b4ac-54d4939c4e14)

**Comment by imtipi** [comment]
![image](https://github.com/ethereum-optimism/developers/assets/27230181/87426fa0-1632-4284-b853-03b99a93aecc)

search country name(china / 中国),not the code

**Reply by megtog** [reply]
Thanks!  But now it says 
> Invalid phone number

I deleted my phone number in the screenshot.  When the phone number is typed it still says "Invalid phone number" (Actually when I clicked send, there would be a captcha.  After I'm done with the captcha, it would say "Invalid phone number".  Then if I type the phone number and click again, there would be a new captcha)

Would be great if this phone number process could be disabled.

![image](https://github.com/ethereum-optimism/developers/assets/106442058/7656611f-4d02-4dbe-a296-006a4106da62)

**Reply by imtipi** [reply]
try switch language to chinese,i also got invalid number while using traditional Chinese,but works with simplified Chinese
a reference I found: https://www.bilibili.com/read/cv21552660/

> Would be great if this phone number process could be disabled.

due to large number of spam/scam,I don't think team will disable this

**Comment by 1195597989** [comment]
Maybe you need to use a Hong Kong node in the clash or v2ray
